dashboard.csss-ci.com/server/api/controllers/rest/v1/Proposals.php

```diff
@@ -81,6 +95,9 @@ public function index_post() {
         }
     }
 
+    /**
+     * Inserts quotation entry to proposals model database if all necessary fields are valid, otherwise throws an error  
```

Comment:
This recommendation is not necessary, but I think it might make more sense if you wrote "records a quotation to the proposals...."

I think that it might make more sense to the average person if you used "records" instead of insert"